recipe/meta.yaml

```diff
@@ -12,34 +12,33 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install --ignore-installed --no-deps .
+  skip: true  # [py<26 or py==30 or py==31 or py==32 or py==33]
```

Comment:
I don't see a point of putting these constraints here, since we don't build anything below Python 3.7. But if you insist, wouldn't it be nicer to write `py<26 or py>=30 and py<=33`?

Reply:
There might be some value on out pinnings being as close as possible to the upstream requirements. but I could see why there might be some value on just simply pinning to packages that we normally build.